src/utils/whitelistApis.ts

```diff
@@ -2724,6 +2724,16 @@ export const API_LIST = {
                 ROLE.PUBLIC,
               ],
           },
+        '/proxies/v8/program/v2/admin/bulkEnroll': {
+            checksNeeded: [CHECK.ROLE],
+            // tslint:disable-next-line: object-literal-sort-keys
+            ROLE_CHECK: [
+                ROLE.PROGRAM_COORDINATOR,
+                ROLE.CBP_ADMIN,
+                ROLE.MDO_ADMIN,
+                ROLE.CONTENT_REVIEWER,
```

Comment:
Only Content_Creator and Program_Coordinator roles should be added for this API

Reply:
resolved